docs/api.md

```diff
@@ -2200,8 +2201,14 @@ The method runs `element.querySelectorAll` within the page. If no elements match
 
 The method evaluates the XPath expression relative to the elementHandle. If there's no such element, the method will resolve to `null`.
 
+#### elementHandle.accessibleName()
+- returns: <[Promise]<?[string]>>
+
+This method computed the acessible name of the element that would exposed to a screen reader.
```

Comment:
computed -> computes?
acessible -> accessible?
would exposed -> would be exposed?